lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/CrowdGroupMembersDAO.java

```
@@ -0,0 +1,151 @@
+package org.sagebionetworks.repo.model.dbo.dao;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.sagebionetworks.authutil.AuthenticationException;
+import org.sagebionetworks.authutil.CrowdAuthUtil;
+import org.sagebionetworks.repo.model.DatastoreException;
+import org.sagebionetworks.repo.model.GroupMembers;
+import org.sagebionetworks.repo.model.GroupMembersDAO;
+import org.sagebionetworks.repo.model.UserGroup;
+import org.sagebionetworks.repo.model.UserGroupDAO;
+import org.sagebionetworks.repo.web.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * 
+ * Implementation of GroupMembersDAO which gets information from Crowd
+ * This class will have a limited life span (1 or 2 weeks) while transitioning user groups away from Crowd
+ *
+ */
+public class CrowdGroupMembersDAO implements GroupMembersDAO {
+	
+	@Autowired
+	private UserGroupDAO userGroupDAO;
+	
+	@Override
+	public GroupMembers getMembers(String principalId)
+			throws DatastoreException {
+		
+		GroupMembers members = new GroupMembers();
+		members.setId(principalId);
+		UserGroup group;
+		try {
+			group = userGroupDAO.get(principalId);
+		} catch (NotFoundException e) {
+			throw new DatastoreException(e);
```

this should throw a not found